examples/interaction/js_callbacks/color_sliders.py

```diff
@@ -2,141 +2,74 @@
 ``CustomJS`` callbacks that can update a plot.
 
 .. bokeh-example-metadata::
-    :apis: bokeh.plotting.figure.rect, bokeh.plotting.figure.text, bokeh.layouts.column, bokeh.layouts.row, bokeh.models.sources.ColumnDataSource, bokeh.models.callbacks.CustomJS, bokeh.models.widgets.sliders.Slider, bokeh.themes.Theme # noqa: E501
+    :apis: bokeh.plotting.figure.text, bokeh.layouts.column, bokeh.models.sources.ColumnDataSource, bokeh.models.callbacks.CustomJS, bokeh.models.widgets.sliders.Slider, bokeh.themes.Theme # noqa: E501
     :refs: :ref:`ug_interaction_js_callbacks_customjs`
     :keywords: hover, javascript callback, theme, tooltip
 
 '''
-import colorsys
-
-import yaml
-
-from bokeh.layouts import column, row
+from bokeh.layouts import column
 from bokeh.models import ColumnDataSource, CustomJS, Slider
 from bokeh.plotting import curdoc, figure, show
 from bokeh.themes import Theme
 
+R, G, B = (75, 125, 125)
 
-# for plot 2: create colour spectrum of resolution N and brightness I, return as list of decimal RGB value tuples
```

Comment:
removed plot 2 as it has nothing at all to do with sliders and adds a large amount of weight to the example